src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff/DiffGenerator.cs

```diff
@@ -0,0 +1,610 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using DiffPlex.DiffBuilder;
+using DiffPlex.DiffBuilder.Model;
+using Microsoft.DotNet.ApiSymbolExtensions.Logging;
+using Microsoft.DotNet.ApiSymbolExtensions;
+using System.Diagnostics;
+using Microsoft.DotNet.ApiSymbolExtensions.Filtering;
+using Microsoft.DotNet.GenAPI;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.DotNet.ApiDiff;
+
+/// <summary>
+/// Generates a markdown diff of two different versions of the same assembly.
+/// </summary>
+public sealed class DiffGenerator
+{
+    private readonly ILog _log;
+    private readonly bool _addPartialModifier;
+    private readonly bool _hideImplicitDefaultConstructors;
+    private readonly ISymbolFilter _symbolFilter;
+    private readonly ISymbolFilter _attributeSymbolFilter;
+    private readonly SyntaxTriviaList TwoSpacesTrivia;
+    private readonly SyntaxList<AttributeListSyntax> EmptyAttributeList;
```

Comment:
These two fields should start with an `_` to match the other fields.

Reply:
Done.